templates/user/settings/applications.tmpl

```diff
@@ -40,222 +48,43 @@
 			<h5 class="ui top header">
 				{{.locale.Tr "settings.generate_new_token"}}
 			</h5>
-			<p>{{.locale.Tr "settings.new_token_desc"}}</p>
-			<form class="ui form ignore-dirty" action="{{.Link}}" method="post">
+			<form id="scoped-access-form" class="ui form ignore-dirty" action="{{.Link}}" method="post">
 				{{.CsrfTokenHtml}}
 				<div class="field {{if .Err_Name}}error{{end}}">
 					<label for="name">{{.locale.Tr "settings.token_name"}}</label>
 					<input id="name" name="name" value="{{.name}}" autofocus required maxlength="255">
 				</div>
-				<!--Temporarily disable-->
-				<details class="gt-hidden ui optional field">
+				<div class="field">
+					<label>{{.locale.Tr "settings.repo_and_org_access"}}</label>
+					<label style="cursor: pointer">
+						<input class="enable-system" type="radio" name="scope" value="{{$.AccessTokenScopePublicOnly}}" style="margin: 4px 5px 0 0">
+						{{.locale.Tr "settings.permissions_public_only"}}
+					</label>
+					<label style="cursor: pointer">
```

Comment:
Please do not use inline styles.
Use or define a helper class instead.
I think there is already a helper for that.

Reply:
was resolved